uxarray/core/dataarray.py

```diff
@@ -57,3 +57,9 @@ def uxgrid(self):
     @uxgrid.setter
     def uxgrid(self, ugrid_obj):
         self._uxgrid = ugrid_obj
+
+    @property
+    def topology(self):
+        """Returns a reference to the internal grid dataset that contains Grid
+        Topology Variables."""
+        return self._uxgrid._ds
```

Comment:
Please see my [comment](https://github.com/UXARRAY/uxarray/issues/324#issuecomment-1561532327) in the issue #324